src/wp-includes/formatting.php

```diff
@@ -2035,7 +2035,15 @@ function sanitize_file_name( $filename ) {
 	}
 
 	if ( $utf8_pcre ) {
-		$filename = preg_replace( "#\x{00a0}#siu", ' ', $filename );
+		/**
+		 * Replace all whitespace characters with a basic space (U+0020).
+		 *
+		 * Characters in the White_Space category are listed with “Zs” in
+		 * their entry in the UnicodeData file maintained at the linked URL.
```

Comment:
[I think the correct name for this property is "space separator," not "white space".
](https://www.unicode.org/reports/tr44/#General_Category_Values)

```suggestion
		 * Replace all whitespace characters with a basic space (U+0020).
		 *
		 * Characters in the  “Space_Separator” category are listed with “Zs” in
		 * their entry in the UnicodeData file maintained at the linked URL.
```

> `Zl`  Line separator 
> `Zp`  Paragraph separator 
> `Zs`  Space separator

Reply:
I typed with the wording; would appreciate your thoughts

Reply:
after @adamziel raised his own similar questions I expanded the comment with more links